examples/mspaint.py

```diff
@@ -8,15 +8,36 @@
 The example shows how to work with MS Paint application. It opens
 JPEG image and resizes it using "Resize and Skew" dialog.
 """
-
+import logging
+import sys
+from pywinauto import actionlogger
 from pywinauto import Application
 
+is_logging_enabled = False
+file_name_log = ''
+if len(sys.argv) >= 2 and len(sys.argv) <= 3:
+    if sys.argv[1] == '--log':
+        is_logging_enabled = True
+        file_name_log = 'log_mspaint.txt'
+        if len(sys.argv) == 3:
+            file_name_log = sys.argv[2]
+    else:
+        print("""Usage: python mspaint.py --log <filename>\n
+             --log - enable logging to a file (an optional parameter)\n
+             <filename> - log file name (an optional parameter, default: """)
+        exit()
+
+if is_logging_enabled:
+    actionlogger.enable()
+    logger = logging.getLogger('pywinauto')
+    logger.handlers[0].stream = open(file_name_log, 'w')
```

Comment:
Why not using standard class `FileHandler` explicitly? It should be more clear and you don't have to close anything at the end (the class will do this job for you).